dev/com.ibm.websphere.appserver.jsfContainer-2.3/resources/l10n/com.ibm.websphere.appserver.jsfContainer-2.3.properties

```diff
@@ -0,0 +1,20 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables CDI integrations for web applications that use the Java Server Faces \
+ (JSF) 2.3 framework. This feature only includes container integration code for JSF, and does \
+ not include any JSF interfaces or implementation.  To use this feature, JSF interfaces and implementation \
+ must be packaged within web applications that use JSF.
```

Comment:
Looks like some extra spaces here. Other than that, I would name the feature instead of "This feature..." and "needs to be packaged" over "must".

Reply:
Thanks @bcpswope for the feedback. We want to keep the jsfContainer-2.3.properties file as close to jsfContainer-2.2 version as possible.